quinn-proto/src/connection/packet_crypto.rs

```diff
@@ -0,0 +1,70 @@
+use tracing::{debug, trace};
+
+use crate::connection::spaces::PacketSpace;
+use crate::crypto::{HeaderKey, PacketKey};
+use crate::packet::{Packet, PartialDecode};
+use crate::token::ResetToken;
+use crate::RESET_TOKEN_SIZE;
+
+pub(super) struct UnprotectHeaderResult {
```

Comment:
Style nit: this should go below `unprotect_header()`.

Reply:
The idea is to put it after the `fn`, right? I'll could do the same with `DecryptPacketResult`

Reply:
Yup.